channeldb/graph.go

```
@@ -1023,6 +1023,10 @@ func (c *ChannelGraph) AddChannelEdge(edge *models.ChannelEdgeInfo,
 	}
 
 	for _, f := range op {
+		if f == nil {
```

I think if we wanna be safu we should return an error to the callsite

---

Made it an error